src/lando/api/legacy/workers/automation_worker.py

```diff
@@ -229,18 +231,20 @@ def run_automation_job(self, job: AutomationJob) -> bool:
         return True
 
     def run_automation_checks(
-        self, scm: AbstractSCM, commits: list[CommitData]
+        self, scm: AbstractSCM, requester_email: str, commits: list[CommitData]
     ) -> list[str]:
         """Run landing checks on the stack of commits generated by the job.
 
         Returns a list of error messages.
         """
-        patch_helpers = []
-        # Create PatchHelpers from commits
-        for commit in commits:
-            patch_helpers.append(scm.get_patch_helper(commit.hash))
+        patch_helpers = filter(
+            lambda x: x is not None,
```

Comment:
Using a variable name other that `x` would be better here, and also would be good to add a comment as to when the argument is expected to be None.